pom.xml

```diff
@@ -335,6 +353,9 @@
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
+        <configuration>
+          <layout>ZIP</layout>
```

Comment:
Warum müssen wir das layout anpassen? Ich hab irgendwie im Hinterkopf das man das machen muss, finde aber keine Doku mehr dazu. Hast du mir da was?

Hier steht davon nichts https://docs.spring.io/spring-boot/docs/current/reference/html/executable-jar.html#appendix.executable-jar.property-launcher

Reply:
siehe https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging

Reply:
ich weiss nicht warum wir das brauchen, da wir über den loader ja lediglich den classpath erweitern und wir die uv ja nicht neu zusammenbauen. Ich denke das kann raus?

Reply:
"ZIP (alias to DIR): similar to the JAR layout using PropertiesLauncher." dafür brauchen wir es